### ISSUE TITLE
ROCM conflicts with openmpi

### ISSUE THREAD

**Issue by PhilipDeegan** [RadeonOpenCompute/ROCm#1017]
I'm not trying to use either of my GPUs, is there a simple way to prevent them being picked up via mpirun?

```
mpirun -np 2 echo "foo"
Memory access fault by GPU node-2 (Agent handle: 0x557153bfcf50) on address (nil). Reason: Unknown.
Aborted
```

**Comment by Dekken** [comment]
I had compiled OpenMPI 4.0.(2?) locally when this began.
I didn't happen with OpenMPI 3.x from apt (buster-backports) or MPICH (latest from source) 

Removing rocm* from via apt allowed mpirun to complete, not exactly ideal!

**Comment by ROCmSupport** [comment]
Thanks @PhilipDeegan 
This issue is fixed and not observed with the latest ROCm 4.1.
Hence recommend you to try with the same.
Feel free to open a new issue, for any, for quick resolution.
Thank you.
